docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -85,37 +83,40 @@ Using a custom namespace solves problems later on because you do not have to ove
 1. Install Grafana Mimir using the Helm chart:
 
    ```bash
-   helm -n <namespace> install <release-name> grafana/mimir-distributed -f custom.yaml
+   helm -n mimir-test install mimir grafana/mimir-distributed -f custom.yaml
    ```
 
-1. Check the statuses of the Mimir services:
+   > **Note:**: the output of the command contains the write and read URLs necessary for the following steps.
+
+1. Check the statuses of the Mimir pods:
 
    ```bash
-   kubectl -n <namespace> get pod
+   kubectl -n mimir-test get pod
    ```
 
    The results look similar to this:
 
    ```bash
-   kubectl -n dev get pod
-   NAME                                                       READY   STATUS      RESTARTS   AGE
-   <release-name>-mimir-nginx-69fd969c-g6bkn                  1/1     Running     0          159m
-   <release-name>-minio-5757f44456-dz7xx                      1/1     Running     0          159m
-   <release-name>-mimir-distributed-make-bucket-job-f5gbj     0/1     Completed   0          159m
-   <release-name>-mimir-distributor-7db5f7c8bc-444sd          1/1     Running     0          148m
-   <release-name>-mimir-overrides-exporter-79b475c98d-sqzxx   1/1     Running     0          148m
-   <release-name>-mimir-querier-5cb58b7b9c-zwnr6              1/1     Running     0          148m
-   <release-name>-mimir-query-frontend-74d666bcd5-4dcwh       1/1     Running     0          148m
-   <release-name>-mimir-ruler-745fff6c6d-8swsv                1/1     Running     0          148m
-   <release-name>-mimir-alertmanager-0                        1/1     Running     0          148m
-   <release-name>-mimir-compactor-0                           1/1     Running     0          148m
-   <release-name>-mimir-ingester-1                            1/1     Running     0          147m
-   <release-name>-mimir-ingester-2                            1/1     Running     0          147m
-   <release-name>-mimir-store-gateway-0                       1/1     Running     0          146m
-   <release-name>-mimir-ingester-0                            1/1     Running     0          144m
+   kubectl  -n mimir-test get pod
+   NAME                                            READY   STATUS      RESTARTS   AGE
+   mimir-minio-78b59f5569-fhlhs                    1/1     Running     0          2m4s
+   mimir-nginx-74f8bff8dc-7kr7z                    1/1     Running     0          2m5s
+   mimir-mimir-distributed-make-bucket-job-z2hc8   0/1     Completed   0          2m4s
```

Comment:
Looks like you missed this one (extra mimir-)

Reply:
```suggestion
   mimir-distributed-make-bucket-job-z2hc8   0/1     Completed   0          2m4s
```

Reply:
it's a tiny bug : https://github.com/grafana/helm-charts/issues/1345

Reply:
but we'll fix it in the doc anyway